src/test/ui/associated-type/associated-type-projection-from-multiple-supertraits.stderr

```diff
@@ -20,19 +35,28 @@ LL |     type Color;
    |     ----------- ambiguous `Color` from `Box`
 ...
 LL | fn dent_object<COLOR>(c: dyn BoxCar<Color=COLOR>) {
-   |                                     ^^^^^^^^^^^ ambiguous associated type `Color`
+   |                              ^^^^^^^^^^^^^^^^^^^ ambiguous associated type `Color`
+   |
+help: use fully qualified syntax to disambiguate
+   |
+LL | fn dent_object<COLOR>(c: dyn <BoxCar as Box>::Color) {
+   |                              ^^^^^^^^^^^^^^^^^^^^^^
+help: use fully qualified syntax to disambiguate
+   |
+LL | fn dent_object<COLOR>(c: dyn <BoxCar as Vehicle>::Color) {
+   |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-error[E0191]: the value of the associated type `Color` (from the trait `Vehicle`) must be specified
-  --> $DIR/associated-type-projection-from-multiple-supertraits.rs:23:26
+error[E0191]: the value of the associated type `Color` (from trait `Vehicle`) must be specified
+  --> $DIR/associated-type-projection-from-multiple-supertraits.rs:25:30
    |
 LL |     type Color;
    |     ----------- `Color` defined here
 ...
 LL | fn dent_object<COLOR>(c: dyn BoxCar<Color=COLOR>) {
-   |                          ^^^^^^^^^^^^^^^^^^^^^^^ associated type `Color` must be specified
+   |                              ^^^^^^^^^^^^^^^^^^^ help: specify the associated type: `BoxCar<Color=COLOR, Color = Type>`
```

Comment:
It'd be nice to handle this case in a more graceful manner, but realistically we should rely on the prior error giving context on what to do here. At best we should silence E0191 when E0221has been emitted for the same span.